driver-core/src/main/java/com/datastax/driver/core/Configuration.java

```diff
@@ -49,7 +51,8 @@ public Configuration() {
              new PoolingOptions(),
              new SocketOptions(),
              new MetricsOptions(),
-             new QueryOptions());
+             new QueryOptions(),
+             new NettyCustomizer());
```

Comment:
Since NettyCustomizer doesn't really have any state, should we have a default static instance?   Much like DefaultRetryPolicy has an INSTANCE that gets reused?